scripts/migration/fix_version_defaults_v2.py

```
@@ -0,0 +1,147 @@
+                        if "default_factory" not in context:
+                            needs_fix = True
+                            break
+
+                if needs_fix:
+                    fixes = fix_file(filepath)
+                    if fixes > 0:
+                        rel_path = filepath.relative_to(Path(__file__).parent)
+                        print(f"  ✓ {rel_path}: {fixes} fix(es)")
+                        total_fixes += fixes
+                        files_fixed += 1
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)